### PR TITLE
updating proxy sha everywhere

### PR DESCRIPTION
**Pull request by rshriram**
The updated proxy has support for WebSockets, more flexible zipkin traces (compatible with Jaeger, etc.).

### CODE REVIEW

File: WORKSPACE [hunk 1]
```diff
@@ -421,7 +421,7 @@ go_repository(
 ##
 
 # Change this and the docker/Dockerfile.proxy* files together
```
Comment (ldemailly) [inline WORKSPACE]: maybe also add here that we currently get the sha from the jenkins post submit ?

**Comment by rshriram** [comment]
I can approve my own PR? Thank you bot. Life is good.

**Comment by ldemailly** [comment]
you still need a reviewer - prow bot is distorting the meaning of approve a bit and ties it to owners

**Comment by qiwzhang** [comment]
I tested this integration test with https://github.com/istio/proxy/pull/427.  Envoy still crashes at mixerclient global dictionary shrink which is not protected.  If mixer is older version with fewer global dictionary, you will run into this issue.

I am working the fix.

**Comment by kyessenov** [comment]
@rshriram you might to update mixer SHA in driver.go to pick a later mixer. Did mixer break protocol again?

**Comment by ldemailly** [comment]
let's get envoy 1.4.0 too?

**Comment by rshriram** [comment]
force merging.. all tests passing!! yay

**Pull request merged**